src/run.ts

```diff
@@ -105,54 +107,54 @@ const LINUX = 'Linux'
 const MAC_OS = 'Darwin'
 const WINDOWS = 'Windows_NT'
 const ARM64 = 'arm64'
-export function getHelmDownloadURL(version: string): string {
+export function getHelmDownloadURL(baseURL: string, version: string): string {
    const arch = os.arch()
    const operatingSystem = os.type()
 
    switch (true) {
       case operatingSystem == LINUX && arch == ARM64:
          return util.format(
-            'https://get.helm.sh/helm-%s-linux-arm64.zip',
+            `${baseURL}/helm-%s-linux-arm64.zip`,
```

Comment:
we should use https://nodejs.org/api/url.html to handle merging the path and baseurl since we now accept user input. Will make this more resilient.

Reply:
I totally agree and will send a commit soon.

@OliverMKing thanks for the first review.

Reply:
@OliverMKing the PR is ready for a second review